ResourceStatusSystem/PolicySystem/Actions/SlackAction.py

```diff
@@ -0,0 +1,122 @@
+""" SlackAction
+
+  This action posts the status change to a Slack channel
+  It uses Slack Webhooks.
+  Also compatible with Mattermost, which is an open source alternative to Slack.
+
+  To create a webhook URL refer the following :
+  Slack : https://api.slack.com/incoming-webhooks
+  Mattermost : https://docs.mattermost.com/developer/webhooks-incoming.html
+
+  Add the webhook URL to dirac.cfg at Operations/[]/ResourceStatus/Config/Slack
+
+  example:
+  Operations/
```

Comment:
There needs to be an empty line here, and the whole block needs to be indented

Reply:
Done